.pre-commit-config.yaml

```diff
@@ -31,6 +31,10 @@ repos:
           "--filter-files",
         ]
       files: ^(pennylane/|tests/)
+-   repo: https://github.com/gauge-sh/tach-pre-commit
+    rev: v0.13.1
+    hooks:
+    -   id: tach
```

Comment:
We don't need to specify the path to the toml file for this similarly to the `.pylintrc` file for pylint?

Reply:
Yep. It is in the root of the repo, so will (AKA should) work.